sbws/core/scanner.py

```diff
@@ -557,7 +557,6 @@ def gen_parser(sub):
 def main(args, conf):
     if conf.getint('scanner', 'measurement_threads') < 1:
         fail_hard('Number of measurement threads must be larger than 1')
-
```

Comment:
Please don't include formatting changes in a commit that contains code changes. If this reformatting needs to happen, it should go in a separate commit.

Reply:
Was a mistake. Fixed